game/src/game/player.rs

```diff
@@ -29,26 +60,66 @@ pub struct PlayerGfxBundle {
     sprite: SpriteSheetBundle,
     animation: SpriteAnimationBundle,
 }
-
+//where is this added/spawned, currently debug spawned only
 #[derive(Component, Default)]
 pub struct PlayerBlueprint;
 
-#[derive(Component, Default)]
-pub struct PlayerGent;
+#[derive(Component)]
+pub struct PlayerGent {
+    e_gfx: Entity,
+}
 
 #[derive(Component, Default)]
 pub struct PlayerGfx;
```

Comment:
Speaking of which, perhaps we should add an Entity for the gent here too? We could have these two components (PlayerGent and PlayerGfx) serve as the canonical place for linking the two entities.

(And remove it from `struct TransformGfxFromGent` in gent.rs)

Reply:
Yeah I think that makes sense actually. Thats where I looked for it at first too before I found it in `TransformGfxFromGent`